lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -308,7 +327,7 @@ const Reportback = {
         'isListItem': true,  // @TODO: probably a better way to address this... :insert shruggy emoji here:
         'reactions': {
           enabled: setting('dsKudosReactions.enabled'),
-          data: this.getReactions(items[i].kudos.data),
+          data: setting('dsKudosReactions.likedByCurrentUser') ? this.getReactions(items[i].kudos, true) : this.getReactions(items[i].kudos.data, false),
         }
```

Comment:
ohh you mean instead of returning a boolean for `liked_by_current_user`, return the action `reaction` object in the Rogue response?